### PR TITLE
Workingbranch

### PR DESCRIPTION
**Pull request by timurbuch**

### CODE REVIEW

File: icons/css.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="32px" height="32px" viewBox="0 0 32 32" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns">
+    <!-- Generator: Sketch 3.0.3 (7891) - http://www.bohemiancoding.com/sketch -->
+    <title>icon 77 document file css</title>
+    <desc>Created with Sketch.</desc>
```
Comment (macfrei) [inline icons/css.svg:5]: You could change the title and description here to what this svg actually is for.

File: index.html [hunk 2]
```diff
@@ -6,58 +6,65 @@
...
         </nav>
     </header>
-    <figure>
-        <img src="img/Cat.jpeg"
-        alt="placeholder picture of a cute cat">
-        <br />
+    <div class="hero">Responsive Front End Development</div>
+    <!--<figure>
+        <img src="img/Cat.jpeg" alt="placeholder picture of a cute cat">
         <figcaption>A placeholder picture of a beautiful cat</figcaption>
```
Comment (macfrei) [inline index.html:25]: Usually we don't leave commented out code in and don't merge it into the main branch. That one should be as clean as possible 🙂

File: index.html [hunk 3]
```diff
@@ -6,58 +6,65 @@
...
-            <li>Programming</li>
-            <li>Good Food</li>
-            <li>Traveling</li>
-            <li>Tennis</li>
-            <li>History</li>
+            <li class="card"><img src="icons/laptop.svg">Programming</li>
+            <li class="card"><img src="icons/food.svg">Good Food</li>
+            <li class="card"><img src="icons/vacay.svg">Traveling</li>
+            <li class="card"><img src="icons/tennis.svg">Tennis</li>
+            <li class="card"><img src="icons/books.svg">History</li>
```
Comment (macfrei) [inline index.html:34]: The `alt` attribute is missing in these images!

File: index.html [hunk 4]
```diff
@@ -6,58 +6,65 @@
...
+            <li class="card"><img src="icons/303903961594722075.svg">CLI</li>
+            <li class="card"><img src="icons/html.svg">HTML</li>
+            <li class="card"><img src="icons/css.svg">CSS</li>
+            <li class="card"><img src="icons/js.svg">Javascript</li>
         </ul>
     </section>
     <section id="contact">
 
         <h2>Contact</h2>
         <p>Phone: +49 17694723961
```
Comment (macfrei) [inline index.html:50]: Check out how to display phone numbers in HTML and the `address` element!

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+h1 {
+    align-self: center;
+    justify-self: flex-start;
+    font-weight: bolder;
+    margin-left: 5%;
+}
+nav {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
```
Comment (macfrei) [inline styles.css:34]: This is not needed as this is the default 🙂

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+    margin:7%;
```
Comment (macfrei) [inline styles.css:35]: You need to be careful with percentages though 🙂

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+    font-weight: 800;
+    width:10%;
+    flex-wrap: wrap;
+
+
+    
+}
+a {
+    text-decoration: none;
```
Comment (macfrei) [inline styles.css:44]: Usually in the web links are displayed with an underline. I would add a `border-bottom` if you decide to remove it.

**Review by macfrei — COMMENTED** [review]
Good job! Just added a few comments you might want to consider 🙂

File: index.html (hunk 2 above)
Comment (timurbuch) [reply index.html:25]: Bad practice, hopefully it won't happen again 😃

File: index.html (hunk 3 above)
Comment (timurbuch) [reply index.html:34]: Thanks, will add them immediately.

File: styles.css (hunk 6 above)
Comment (timurbuch) [reply styles.css:35]: Changed it to vw and vh.

File: icons/css.svg (hunk 1 above)
Comment (timurbuch) [reply icons/css.svg:5]: Will do 👍🏾

File: styles.css (hunk 5 above)
Comment (timurbuch) [reply styles.css:34]: Removed 🦾

File: styles.css (hunk 7 above)
Comment (timurbuch) [reply styles.css:44]: Still undecided how to style the web links, but I will consider it.

Comment (macfrei) [reply styles.css:44]: Coolio! And well done! 😎

Comment (timurbuch) [reply styles.css:44]: Thank you 😸

**Pull request merged**